packages/react-component-library/cypress/support/consoleErrors.ts

```diff
@@ -0,0 +1,19 @@
+import { Cypress, cy } from 'local-cypress'
+
+function check() {
+  let stub
+
+  Cypress.on('window:before:load', (win) => {
+    stub = cy.stub(win.console, 'error')
+  })
+
+  Cypress.on('command:end', () => {
+    if (stub) {
```

Comment:
It does seem a little bit spammy when there aren't any errors. Could add in ` && stub.called` here so it doesn't log anything on success (a bit of a cheat of course).